src/include/daos_obj_class.h

```diff
@@ -141,30 +193,15 @@ enum {
 	 * S1 : shards=1, S2 means shards=2, ...
 	 * SX : spreading across all targets within the pool
 	 */
-	OC_S1		= 200,
-	OC_S2,
-	OC_S4,
-	OC_S6,
-	OC_S8,
-	OC_S12,
-	OC_S16,
-	OC_S24,
-	OC_S32,
-	OC_S48,
-	OC_S64,
-	OC_S96,
-	OC_S128,
-	OC_S192,
-	OC_S256,
-	OC_S384,
-	OC_S512,
-	OC_S768,
-	OC_S1K,
-	OC_S2K,
-	OC_S4K,
-	OC_S6K,
-	OC_S8K,
-	OC_SX,
+	OC_S1		= (OR_RP_1 << OC_REDUN_SHIFT) | 1UL,
```

Comment:
could be better if you add a macro like
#define OBJ_CLASS_DEF(redun, grp_nr).     ((redun << OC_REDUN_SHIFT) | grp_nr)

Reply:
Yup, sure.